--- beast-1.8.1/release/Linux/scripts/beast	2015-01-27 14:35:56.000000000 -0800
+++ patch-files/beast	2015-01-27 14:40:39.952934521 -0800
@@ -23,4 +23,5 @@
 fi
 
 BEAST_LIB="$BEAST/lib"
-java -Xms64m -Xmx2048m -Djava.library.path="$BEAST_LIB:/usr/local/lib" -cp "$BEAST_LIB/beast.jar:$BEAST_LIB/beast-beagle.jar" dr.app.beast.BeastMain $*
+BEAGLE_LIB="$BEAGLE_HOME/lib"
+java -Xms64m -Xmx2048m -Djava.library.path="$BEAST_LIB:$BEAGLE_LIB:/usr/local/lib:$LD_LIBRARY_PATH" -cp "$BEAST_LIB/beast.jar:$BEAST_LIB/beast-beagle.jar" dr.app.beast.BeastMain $*
