--- beast-1.8.1/release/Linux/scripts/beast	2015-01-13 15:20:21.000000000 -0800
+++ patch-files/beast	2015-01-13 15:54:13.536465039 -0800
@@ -22,5 +22,5 @@
 	cd "$saveddir"
 fi
 
-BEAST_LIB="$BEAST/lib"
-java -Xms64m -Xmx2048m -Djava.library.path="$BEAST_LIB:/usr/local/lib:$LD_LIBRARY_PATH" -cp "$BEAST_LIB/beast.jar:$BEAST_LIB/beast-beagle.jar" dr.app.beast.BeastMain $*
+BEAGLE_LIB="$BEAGLE_HOME/lib"
+java -Xms64m -Xmx2048m -Djava.library.path="$BEAGLE_LIB:$BEAST_LIB:/usr/local/lib:$LD_LIBRARY_PATH" -cp "$BEAST_LIB/beast.jar:$BEAST_LIB/beast-beagle.jar" dr.app.beast.BeastMain $*
